plugins/providers/oss/provider.go

```diff
@@ -35,26 +36,19 @@ type Policy struct {
 	Statement []PolicyStatement `json:"Statement"`
 }
 
-type OSSClient struct {
-	client     *oss.Client
-	authConfig aliauth.AliAuthConfig
-}
 type provider struct {
 	pv.UnimplementedClient
 	pv.PermissionManager
 	typeName  string
 	encryptor encryptor
-
-	ossClients map[string]OSSClient
```

Comment:
i believe this caching would still be needed for provider_urn specific client, but now we can directly use `map[string]*oss.Client` type